test/main.sh

```diff
@@ -397,6 +397,7 @@ if [ "${1:-"all"}" != "cluster" ]; then
     run_test test_storage_driver_zfs "zfs storage driver"
     run_test test_storage_driver_pure "pure storage driver"
     run_test test_storage_buckets "storage buckets"
+    run_test test_storage_bucket_export "storage buckets export and import"
```

Comment:
`storage_buckets` is part of the `LXD_REQUIRED_TESTS` for some reason @tomponline probably knows.
Maybe `storage_bucket_export` should be part of that list as well?

It shouldn't change much because the early returns in the `test_storage_bucket_export()` are for conditions that should all be met with GitHub CI but still, it might be safer to hard fail if the test is skipped.